test/integration/resultsPageYourLocation.js

```diff
@@ -102,6 +102,8 @@ describe(`The ${yourLocation} results page`, () => {
         const $ = cheerio.load(res.text);
 
         expect($('.results__header--none').text()).to.be.equal('Your location is not in England');
+        expect($('.results__none-content').text()).to
+          .contain('This service only provides information about pharmacies in England.');
```

Comment:
I don't really like testing for the lack of things, however, I think this test would benefit from testing of the lack of the text that is tested for https://github.com/nhsuk/connecting-to-services/pull/393/commits/951b40d7f1af2bd2bf57bf0bb45deaa681667cce#diff-d47eaffdfc3f1a1634cee7b96469fa02R175